packages/dart/sshnoports/bundles/universal.sh

```diff
@@ -79,12 +85,28 @@ usage() {
 	fi
```

Comment:
@XavierChanth should this be `universal.sh` rather than `install.sh`?

Reply:
Good catch